vault/dendron.topic.special-notes.meeting-note.md

```diff
@@ -1,26 +1,38 @@
 ---
 id: x1d18w46rxu34tksct3d4lh
 title: Meeting Note
-desc: 'Meeting notes help you keep track of meetings by providing a structured default to capture notes, attendes and next steps'
-updated: 1658947585935
+desc: 'Meeting notes help you keep track of meetings by providing a structured default to capture notes, attendees and next steps'
+updated: 1661499836098
 created: 1657070789093
 ---
 
 ## Summary
 
 {{fm.desc}}
 
-## Components
+## Details
```

Comment:
remove this section if we don't use it

Reply:
I'm using it to bunch up everything below, but that may be excessive.
The past few doc standardizations I did use this categorization and if you want I can go back and remove them.